### DISCUSSION TITLE
Lab 7: Exercise 7

### DISCUSSION THREAD

**Discussion by kushtrimsogojeva** [giswqs/geog-312#153] [Q&A] [answered]
It is taking to long to load the dataset. I have to cancell it because it is taking way to long. How you managed to complete it? My code:
`url = "https://overturemaps-tiles-us-west-2-beta.s3.amazonaws.com/2024-09-18/buildings.pmtiles"
metadata = leafmap.pmtiles_metadata(url)
print(f"layer names: {metadata['layer_names']}")
print(f"bounds: {metadata['bounds']}")`

**Comment by giswqs** [comment, marked as answer]
`leafmap.pmtiles_metadata()` is not needed for this question.
